bundler/lib/dependabot/bundler/file_fetcher.rb

```diff
@@ -177,8 +177,10 @@ def fetch_path_gemspec_paths
             PathGemspecFinder.new(gemfile: file).path_gemspec_paths
           end.uniq
         end
-      rescue ::Bundler::LockfileError
-        raise Dependabot::DependencyFileNotParseable, lockfile.path
+      rescue ::Bundler::LockfileError => e
+        msg = e.message.strip
```

Comment:
Should we add that `strip` to the initializer of `DependencyFileNotParseable` instead so we don't have to duplicate it all over the place?

Reply:
Yeah good shout, we could maybe do it for all custom dependabot error messages.